Decorator_pattern/app/src/main/java/com/example/decorator_pattern/Beverage.kt

```diff
@@ -0,0 +1,11 @@
+package com.example.decorator_pattern
+
+abstract class Beverage(val nickname : String = "") {
+
+    abstract fun getCost() : Int
```

Comment:
```suggestion
    abstract val cost : Int
```
불변 필드로 작성하는게 적절해보입니다 :)